PLUGINS_CATALOG.md

```diff
@@ -0,0 +1,37 @@
+# Plugin Catalog
+This is the master catalog of plugins for Snap. The plugins in this list may be written by multiple sources. Please examine the license and documentation of each plugin for more information.
+
+## All Plugins
+This file is automatically generated. If you would like to add to the plugin list, [add your plugin to this list](docs/plugins.yml) and it will be added (usually within 24 hours).
+
+<%-
+metadata = Pluginsync::Plugins.metadata
+%w[ collector processor publisher ].each do |type| 
+-%>
+<%= "### #{Pluginsync::Util.plugin_capitalize(type)}s" %>
+
+| Name | Maintainer | Description | CI | Download |
+|------|------------|-------------|----|----------|
+<%-
+  metadata.find_all{|p| p["type"] == type }.sort_by{|p| p["name"]}.each do |p|
+    maintainer = Pluginsync::Util.org_capitalize(p["maintainer"])
+    downloads = []
+    downloads += ["[release](#{p['github_release']})"] if p.include? "github_release"
+    downloads += p["download"]["s3_latest"].collect{|h| "[#{h.keys.first}](#{h[h.keys.first]})" } if p["download"] and p["download"]["s3_latest"]
+-%>
+| [<%= p["name"] %>](<%= p['repo_url'] %>) | <%= maintainer %> | <%= p["description"] %> | <%= Pluginsync::Util.html_list(p["badge"]) -%> | <%= Pluginsync::Util.html_list(downloads) %> |
```

Comment:
When working on https://github.com/intelsdi-x/snap/pull/1231, I liked @IRCody's idea to link to the maintainer. Could we swap that in? Thinking: 

`<%= maintainer %>](<%= p['html_url'] %>` << I think it's `html_url` here.